packages/gatsby/src/utils/webpack-utils.js

```diff
@@ -292,7 +292,7 @@ module.exports = async ({
     let js = (options = {}) => {
       return {
         test: /\.jsx?$/,
-        exclude: vendorRegex,
+        exclude: /core-js/,
```

Comment:
Somehow core-js needs to be excluded because babel will try to polyfill core-js with core-js XD also mini-extra-css-plugin doesn't work well without this. This magically fixes it, i'm not 100% sure so the above comments are just speculation 😄 .

I'll add a comment when it's getting out of draft mode.

Reply:
> will try to polyfill core-js with core-js 😂